src/pages/study-room/study-room.tsx

```diff
@@ -19,6 +26,18 @@ const StudyRoom = () => {
 
   const participants = curParticipants.length > 0 ? curParticipants : participantSummaries;
 
+
+  useEffect(() => {
+    const handlePopState = () => openDialog();
+    window.addEventListener('popstate', handlePopState);
```

Comment:
래핑하는 handlePopState 함수를 만들기보다 이렇게 그대로 표현해줘도 괜찮지 않을까요?
```suggestion
    window.addEventListener('popstate', openDialog);
```

Reply:
빼다고 했는데 깜빡했네요 ㅠㅠ

Reply:
여기도 동일합니다!